e2e/nomostest/reset.go

```diff
@@ -478,34 +479,59 @@ func initialCommit(nt *NT, repoType gitproviders.RepoType, nn types.NamespacedNa
 	}
 }
 
-// TailReconcilerLogs starts tailing a reconciler's logs.
-// The logs are stored in memory until either the context is cancelled or the
-// kubectl command exits (usually because the container exited).
+// TailCSPodsAndReconcilerLogs starts tailing CS Pods and the reconciler's logs.
+// The pods and logs are stored in memory until either the context is cancelled
+// or the kubectl command exits (usually because the container exited).
 // This allows capturing logs even if the reconciler is deleted before the
 // test ends.
-// The logs will only be printed if the test has failed when the command exits.
-// Run in an goroutine to capture logs in the background while deleting RSyncs.
-func TailReconcilerLogs(ctx context.Context, nt *NT, reconcilerNN types.NamespacedName) {
-	out, err := nt.Shell.KubectlContext(ctx, "logs",
-		fmt.Sprintf("deployment/%s", reconcilerNN.Name),
-		"-n", reconcilerNN.Namespace,
-		"-c", reconcilermanager.Reconciler,
-		"-f")
-	nt.T.Logf("finished tailing logs for %s", reconcilerNN.String())
-	// Expect the logs to tail until the context is cancelled, or exit early if
-	// the reconciler container exited.
-	if err != nil && err.Error() != "signal: killed" {
-		// We're only using this for debugging, so don't trigger test failure.
-		nt.T.Logf("Failed to tail logs from reconciler %s: %v", reconcilerNN, err)
-	}
-	// When TailReconcilerLogs is called async, it's possible that the test fails
+// The pods and logs will only be printed if the test has failed when the
+// command exits.
+// Run in a goroutine to capture logs in the background while deleting RSyncs.
+func TailCSPodsAndReconcilerLogs(ctx context.Context, nt *NT, reconcilerNNs ...types.NamespacedName) {
+	var logs []string
+	// Tail CS pods
```

Comment:
this is just listing & describing pods, not tailing, them.

Reply:
As Sam pointed out, we will need N+1 separate goroutines to list and describe pods and tail reconciler logs. I'll create separate functions with more accurate names.

Reply:
Renamed `testPods` to `listAndDescribePods`